admin/src/views/Stats.vue

```diff
@@ -409,13 +417,41 @@ export default Vue.extend({
       .collection('DH6')
       .doc('statistics')
       .onSnapshot((doc: DocumentSnapshot) => {
-        console.log(doc.data());
         if (doc.exists) {
           this.statistics = doc.data();
           this.setAllData();
         }
       });
-    this.initAgeChart();
+    // this.initAgeChart();
+  },
+  computed: {
+    numHackathons: function() {
+      return this.formatChartData(['applicationStats', 'hackathons']);
+    },
+    universities: function() {
+      return this.formatChartData(['applicationStats', 'universities'], { sort: true, limit: 10 });
+    },
+    majors: function() {
+      return this.formatChartData(['applicationStats', 'majors'], { sort: true, limit: 10 });
+    },
+    schoolYears: function() {
+      return this.formatChartData(['applicationStats', 'school_years'], { sort: true });
+    },
+    shirtSizes: function() {
+      return this.formatChartData(['applicationStats', 'shirt_sizes'], { sort: true });
+    },
+    discoveredBy: function() {
+      return this.formatChartData(['applicationStats', 'discovery'], { sort: true });
+    },
+    dietaryRestrictions: function() {
+      return this.formatChartData(['applicationStats', 'dietary_restrictions'], { sort: true, limit: 10 });
+    },
+    comingFrom: function() {
+      return this.formatChartData(['applicationStats', 'travelling_from'], { sort: true });
+    },
+    workshops: function() {
+      return this.formatChartData(['applicationStats', 'workshops'], { sort: true });
+    },
```

Comment:
Planning on removing these computed properties in favour of defining chart-related information in _data.js_, similar to how my/ does it.